phys2bids/tests/test_integration.py

```diff
@@ -368,10 +388,10 @@ def test_integration_info(samefreq_short_txt_file):
     subprocess.run(command_str, shell=True, check=True)
 
     # Check that plot all file is generated
-    assert os.path.isfile('Test_belt_pulse_samefreq_short.png')
+    assert os.path.isfile(os.path.join(test_outdir, 'bids_ignore/Test_belt_pulse_samefreq_short.png'))
```

Comment:
Same concept here: since `os.path.join()` accepts more than two arguments, you can express this as:
    `assert os.path.isfile(os.path.join(test_outdir, 'bids_ignore','Test_belt_pulse_samefreq_short.png'))`